src/main/java/org/mastodon/mamut/tomancak/sort_tree/SortTree.java

```diff
@@ -60,6 +60,11 @@ public static void sortExternIntern( Model model, Collection<Spot> vertices, Col
 		sort( model, vertices, new ExternInternOrder( model.getGraph(), centerSpots ) );
 	}
 
+	public static void sortCellLifetime( Model model, Collection<Spot> vertices )
```

Comment:
Not related to this PR, but:

Utility classes, which are collections of static members, are not meant to be instantiated. Even abstract utility classes, which can be extended, should not have public constructors.
Java adds an implicit public constructor to every class which does not define at least one explicitly. Hence, at least one non-public constructor should be defined.

Reply:
I know but I often forget to add the private constructor. Thank you for spotting this.